beater/java_attacher/java_attacher_test.go

```diff
@@ -79,7 +85,7 @@ func TestBuild(t *testing.T) {
 	want := filepath.FromSlash("/usr/bin/java -jar ./java-attacher.jar") +
 		" --continuous --log-level debug --download-agent-version 1.25.0 --exclude-user root --include-main MyApplication " +
 		"--include-main my-application.jar --include-vmarg elastic.apm.agent.attach=true " +
-		"--config server_url=http://localhost:8200 --config service_name=my-cool-service"
+		"--config server_url=http://localhost:8200"
 
 	cmdArgs := strings.Join(cmd.Args, " ")
 	assert.Equal(t, want, cmdArgs)
```

Comment:
Just an idea that may simplify testing and ordering issues: instead of asserting the whole expected string, you may also assert that certain sub-strings are contained. Such as

```go
assert.Contains(t, "--config server_url=http://localhost:8200", cmdArgs)
assert.Contains(t, "--config service_name=my-cool-service", cmdArgs)
```